pom.xml

```diff
@@ -135,7 +135,7 @@
       <plugin>
         <groupId>net.alchim31.maven</groupId>
         <artifactId>scala-maven-plugin</artifactId>
-        <version>${scala.maven.plugin.version}</version>
+        <version>4.3.0</version>
```

Comment:
What was the reason to get rid of `scala.maven.plugin.version` variable?

Reply:
It appears that other plugins around do not have a dedicated version variable, so I've inlined it.
Is it set from outside by something (by some integration test perhaps)?